config/settings-env/endpoints.json

```diff
@@ -488,6 +502,12 @@
         "method": "POST",
         "timeout": "5s"
       },
+      {
+        "endpoint": "/vdp.connector.v1alpha.ConnectorPublicService/TestSourceConnector",
+        "url_pattern": "/vdp.connector.v1alpha.ConnectorPublicService/TestSourceConnector",
+        "method": "POST",
+        "timeout": "5s"
```

Comment:
@heiruwu not sure if you need to update the timeout to be consistent with the corresponding REST endpoints.

Reply:
agree we should have, updated